components/SignInfoInputComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import styles from "../styles/SigninPage.module.css";
+
+type props = {
+  name: string;
+  changeEvent: (e: any) => void;
+  validText: string;
+  setValidText: React.Dispatch<React.SetStateAction<string>>;
+  pwType?: boolean;
```

Comment:
이런식으로 표현되는게 좋을 것 같네요~
```suggestion
  isPassword?: boolean;
```